content/en/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md

```diff
@@ -274,7 +274,7 @@ on Kubernetes dual-stack support see [Dual-stack support with kubeadm](/docs/set
 1. Optional: Check the cluster health.
 
     ```sh
-    docker run --rm -it \
+    crictl run --rm -it \
```

Comment:
Did you verify using the crictl command? The crictl and docker commands cannot be completely converted, and many commands are not exactly the same.

you can use `crictl  --help` to check or see https://github.com/kubernetes-sigs/cri-tools/blob/master/docs/crictl.md for more details.

Reply:
Hi [mengjiao-liu](https://github.com/mengjiao-liu) yes, realized it will not work. so closed this PR . will create another

Reply:
have created new PR https://github.com/kubernetes/website/pull/39149